doc-content/enterprise-only/vehicle-route-planning/download-ref-imp-proc.adoc

```diff
@@ -9,11 +9,11 @@ You must download and prepare the deployment files before building and deploying
 
 * Product: {PRODUCT_SHORT}
 * Version: {PRODUCT_VERSION}
-. Download *{PRODUCT} {PRODUCT_VERSION} Decision Services Quick Starts* (`kogito-1.5.0-decision-services-quickstarts.zip`).
-//Confirm download name.
-. Download *{PRODUCT} {PRODUCT_VERSION} Maven Repository Kogito and {PLANNER_SHORT} 8 Maven Repository* (`kogito-1.5.0-maven-repository.zip`).
-. Extract the `kogito-1.5.0-maven-repository.zip` file.
-. Copy the contents of the `kogito-1.5.0.GA-maven-repository/maven-repository` subdirectory into the `~/.m2/repository` directory.
+. Download *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* (`{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-decision-services-quickstarts.zip`).
+. Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-decision-services-quickstarts.zip` file.
+. Download *{PRODUCT} {PRODUCT_VERSION} Maven Repository Kogito and {PLANNER_SHORT} 8 Maven Repository* (`{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-kogito-maven-repository.zip`).
+. Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-kogito-maven-repository.zip` file.
+. Copy the contents of the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-kogito-maven-repository/maven-repository` subdirectory into the `~/.m2/repository` directory.
 . Extract the `kogito-1.5.0-decision-services-quickstarts.zip` file.
```

Comment:
Remove that step Extract the "`kogito-1.5.0-decision-services-quickstarts.zip` file."